lib/Distribution/Package/Dhall.hs

```diff
@@ -941,83 +949,159 @@ extension =
 
 
 
-guarded
-  :: Monoid a
-  => Dhall.Type a
-  -> Dhall.Type ( Cabal.CondTree Cabal.ConfVar [Cabal.Dependency] a )
-guarded t =
-  fmap compileGuards
-    $ Dhall.list
-    $ makeRecord
-    $ (,) <$> keyValue "guard" guard <*> keyValue "body" t
+class Factorable a where
```

Comment:
Minor nitpick: I would call this `Diff`

Reply:
`Diff` sounds good, I'll take that suggestion.